tools/jenkins/run_interop.sh

```diff
@@ -37,3 +37,5 @@ export LANG=en_US.UTF-8
 cd $(dirname $0)/../..
 
 tools/run_tests/run_interop_tests.py -l all -s all --cloud_to_prod --cloud_to_prod_auth --use_docker --http2_interop -t -j 12 $@ || true
+tools/run_tests/run_interop_tests.py -l java --use_docker --http2_badserver_interop $@ || true
```

Comment:
Probable explanation of why Jenkins shows success:
The first run of run_interop_tests.py generates report.xml and the other 2 http badserver runs fail already in the build docker image phase, so they don't have a change to create the report.xml - Jenkins ends up seeing report.xml from the first run and if that run succeeded, the run will look green on Jenkins.